### PR TITLE
User Auth Tokens and Profile Model PR

### PR DESCRIPTION
**Pull request by torykling**
Paths seem to be working when we tested them. 

We can register a new user, login to get a token, create a profile, read all profiles, and read, update, and delete a specific profile. 

We wanted to make a pr at this point so we can start testing log in on the front end while other members of the backend team continue to work on other models (trips, topics, etc)

### CODE REVIEW

File: api/Pipfile [hunk 1]
```diff
@@ -10,6 +10,10 @@ pylint = "*"
 django = "*"
 psycopg2-binary = "*"
 autopep8 = "*"
+djangorestframework-simplejwt = "*"
```
Comment (ZakkMan) [inline api/Pipfile]: Are you still using this? If not, can we remove it?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/victoriaklingenstein/.local/share/virtualenvs/CoTrip-TXuPc5lv/bin/python"
+}
```
Comment (ZakkMan) [inline .vscode/settings.json]: Can you delete these settings files?

File: api/accounts/urls.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+from .views import ProfileList, ProfileDetail
+
+urlpatterns = [
+    path('api/auth', include('knox.urls')),
+    path('api/auth/register', RegisterAPI.as_view()),
+    path('api/auth/login', LoginAPI.as_view()),
+    path('api/auth/user', UserAPI.as_view()),
+    path('api/auth/logout', knox_views.LogoutView.as_view(), name='knox_logout'),
+    path('api/profile', ProfileList.as_view(), name='profile_list'),
+    path('api/profile/<int:pk>', ProfileDetail.as_view(), name='profile_detail')
```
Comment (ZakkMan) [inline api/accounts/urls.py]: Maybe the `api` part of these urls is unnecessary?

File: .vscode/settings.json (hunk 2 above)
Comment (torykling) [reply .vscode/settings.json]: Deleted. Not sure where these came from.

File: api/Pipfile (hunk 1 above)
Comment (torykling) [reply api/Pipfile]: Uninstalled.

File: api/accounts/urls.py (hunk 3 above)
Comment (torykling) [reply api/accounts/urls.py]: Removed them

**Pull request merged**